ui/constants/pageTitles.js

```diff
@@ -22,7 +22,7 @@ export const PAGE_TITLE = {
   [PAGES.HELP]: 'Help',
   [PAGES.INVITE]: 'Invite',
   [PAGES.LIBRARY]: 'Purchases',
-  [PAGES.LISTS]: 'Lists',
+  [PAGES.PLAYLISTS]: 'Playlists',
```

Comment:
In the router, I like it that `/$/lists` is still supported for backward compatibility, for users still using old links in their blogposts or whatever.

This side should follow suite, covering both.